### ISSUE TITLE
User can toggle toolbar item, like ordered list

### ISSUE THREAD

**Issue by dmielewczyk** [nhn/tui.editor#812]
## Version
1.4.10

## Development Environment
N/A

## Current Behavior
1. Open editor in WYSIWYG mode
2. Enable ordered list
3. Write some points
4. Click again on ordered list
5. Nothing will happen

## Expected Behavior
1. Open editor in WYSIWYG mode
2. Enable ordered list
3. Write some points
4. Click again on ordered list
5. Ordered list will be deleted (but the text will still be there)

**Comment by northlander** [comment]
I agree with @dmielewczyk - It would be a really nice enhancement to ToastUI 👍

**Comment by backisback** [comment]
As dmielewczyk wrote, the expected behavior is 
- Click again on ordered list
- Ordered list will be deleted (but the text will still be there)

This would be an improvement

**Comment by skrynklarn** [comment]
Can agree with previous comments!

**Comment by prjo** [comment]
Agree would be a good enhancement!

**Comment by seonim-ryu** [comment]
@dmielewczyk @northlander @backisback @skrynklarn @prjo 
Then I'll ask. With a list like this, how do you think it works if you press the toolbar button once more?

![list](https://user-images.githubusercontent.com/18183560/82005974-bd6a9000-96a1-11ea-8dea-759985315c6c.png)

**Comment by backisback** [comment]
@seonim-ryu  A normal behavior is that if you marked the text like the image you present above and press the toolbar, the sublevels of the ul will present it self with blockquote
